README.md

```diff
@@ -74,6 +75,12 @@ table = db.create_table("my_table",
 result = table.search([100, 100]).limit(2).to_pandas()
 ```
 
+## Let's jump to cooking some cool applications using LanceDB.
```

Comment:
Probably remove this section. We'll need a better copy for this.

Reply:
ok